src/components/Dashboard/Pay.tsx

```diff
@@ -138,8 +152,7 @@ export default function Pay() {
                   (payIn === 0 ? parseEther('1') : converter.usdToWei('1')) ??
                     BigNumber.from(0),
                 )}{' '}
-                {tokenSymbol ?? 'tokens'}/
-                <CurrencySymbol currency={payIn} />
+                {tokenSymbol ?? 'tokens'}/{currencyName(payIn)}
```

Comment:
btw, this changes the exchange rate to so token/ETH rather than token/Ξ. I think it's more consistent this way.